orm/class/select.lua

```diff
@@ -540,7 +545,12 @@ local Select = function(own_table)
                 end
 
                 if _set ~= "" then
-                    _update = _update .. " SET " .. _set .. " " .. _where
+                    if #_set_tbl<2 then
+                        _update = _update .. " SET " .. _set .. " " .. _where

```

Comment:
I think that this part is redundant, because ``table.concat`` with one item in the table it will return this value without any comma.

```lua
> print(table.concat({'a'}, ','))
a
```